app/controllers/general.php

```diff
@@ -300,6 +301,35 @@
 
                 Authorization::setRole('role:' . Auth::USER_ROLE_APP);
                 Authorization::setDefaultStatus(false);  // Cancel security segmentation for API keys.
+
+                if (time() > ($key->getAttribute('accessedAt', 0) + (24 * 60 * 60))) {
+                    $key->setAttribute('accessedAt', time());
+                    $sdkValidator = new WhiteList([
+                        'appwrite:nodejs',
+                        'appwrite:deno',
+                        'appwrite:php',
+                        'appwrite:python',
+                        'appwrite:ruby',
+                        'appwrite:go',
+                        'appwrite:java',
+                        'appwrite:dotnet',
+                        'appwrite:dart',
+                        'appwrite:kotlin',
+                        'appwrite:swift'
```

Comment:
@eldadfux What would be a better place to keep these constants ?

Reply:
Not sure, a config might be too much. Let’s leave them here for now

Reply:
Alright 👍